eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -84,6 +84,10 @@
             "matcher": "java-package",
             "match": "/com\\.azure\\.data\\.cosmos(\\..*)?/"
           },
+          {
+            "matcher": "java-package",
+            "match": "/com\\.azure\\.spring\\.cloud\\.config(\\..*)?/"
+          },
```

Comment:
We shouldn't be blanket excluding all classes in this package. Please be specific.

Reply:
@JonathanGiles I was thinking this would be best until we moved the project to have an impl setup. Only 5 of the classes are public apis, so wouldn't any changes to the other classes methods cause an issue?

Reply:
Yes, it would cause an issue...but it also helps to prevent changes to classes that are intended to be public API that reside in the same location (now, or in the future). I'd prefer to err on the side of caution here, because otherwise our reliance on tooling might lead us astray in the future.

Reply:
So would this be updated to
```json
          {
            "matcher": "java",
            "match": "/com\\.azure\\.spring\\.cloud\\.config\\.State?/"
          },
```

Reply:
I observe at the top a line that goes `"class com\\.azure\\.core\\.util\\.Configuration",`. You might be able to do similar here, but of course RevAPI will tell you if you are right or not :-)